doc/ansible/dev/retroactively-owned-resources.md

```diff
@@ -0,0 +1,162 @@
+# Owner References for Existing Resources
+
+Owner references are automatically injected *only during creation of
+resources*. Enabling owner reference injection *will not update objects*
+created while [owner reference injection is
+disabled](./advanced_options.md#turning-off-dependent-watches-and-owner-reference-injection)
+.
+
+This guide will demonstrate how to retroactively set owner references
+for existing resources.
+
+### Owner References and Annotations
+
+Dependent resources *within the same namespace as the owning CR* are
+tracked with the `ownerReference` field.
+
+`ownerReference` structure:
+  * apiVersion: {group}/{version}
+  * kind: {kind}
+  * name: {metadata.name}
+  * uid: {metadata.uid}
+
+**Example:**
+
+```yaml
```

Comment:
what yaml file is this one? Could we add a path as an example?

Reply:
This is just a snippet, there's no file